### DISCUSSION TITLE
Strange canvas reaction

### DISCUSSION THREAD

**Discussion by 90PlusWorks** [szimek/signature_pad#708] [Q&A] [answered]
I'm using the signature pad js to achieve parts of my abc writer and its working fair in portrait mode on my iphone. In landscape I can draw to the top of the canvas if I start below the centre, but I cannot start the line above the centre. 

The demo is on github at https://90plusworks.github.io/ABC-Write/index4.html

I'm guessing I don't understand signature pad well enough to figure what I'm doing wrong. 

Any pointers will be great!

**Comment by 90PlusWorks** [comment]
Its the same in safari, chrome and brave

**Reply by UziTech** [reply]
is this only happening on iPhone?

**Reply by 90PlusWorks** [reply]
I'll ask someone to check on android. On pc it does not rotate but I'll check if it happens in a smaller window. Maybe a div size or position is to blame. My css is less than stellar. I'm layering divs but doing it without complete understanding trial and error

**Reply by 90PlusWorks** [reply]
Also, thanks for the signature pad code. I hacked a little bit in the code to see what changes does to the response. It's possible that I messed up there?

**Reply by 90PlusWorks** [reply]
I'm first going to try and check the resize of a window on the pc on Monday. I'll let you know if I've done that

**Comment by 90PlusWorks** [comment, marked as answer]
This happened due to having multiple overlapping canvas elements, not to do with signature pad code
